prow/kube/client.go

```diff
@@ -37,7 +37,9 @@ const (
 	inClusterBaseURL = "https://kubernetes.default"
 	maxRetries       = 8
 	retryDelay       = 2 * time.Second
-	EmptySelector    = ""
+	requestTimeout   = time.Second * 30
+
+	EmptySelector = ""
```

Comment:
EmptySelector?

Reply:
In [apimachinery](https://github.com/kubernetes/apimachinery/blob/18a564baac720819100827c16fdebcadb05b2d0d/pkg/labels/selector.go#L68) this is called a nothingSelector. I should have probably picked the same name. We have copied [k8s.io/apimachinery/pkg/labels](https://github.com/kubernetes/apimachinery/tree/18a564baac720819100827c16fdebcadb05b2d0d/pkg/labels) in [k8s.io/test-infra/prow/kube/labels](https://github.com/kubernetes/test-infra/tree/master/prow/kube/labels).